pkg/kudoctl/packages/verifier/template/verify_render.go

```diff
@@ -70,3 +60,40 @@ func templateCompilable(pf *packages.Files) verifier.Result {
 
 	return res
 }
+
+func collectPipes(pf *packages.Files) map[string]string {
+	pipes := make(map[string]string)
+	// This grabs keys from ALL pipe tasks for simplicity.
+	// TODO(porridge): feed only these which could be reachable by a given template based
```

Comment:
I generally dislike PRs with new TODOs. It is either: a) a comment about deficiency, b) worth doing immediately or c) a GH issue for tracking

Reply:
Removed, now that we reuse production code.